src/components/common/PaginatedTxns/index.tsx

```diff
@@ -38,6 +40,14 @@ const TxPage = ({
         {isFirstPage && (
           <Box display="flex" flexDirection="column" alignItems="flex-end" mt={['-94px', '-44px']} mb={['60px', 0]}>
             {isQueue ? <BatchExecuteButton items={page.results} /> : <TxFilterButton />}
+            {filter && (
+              <Typography mt={2}>
+                {(page.results.length && page.next
+                  ? `> ${page.results.length} ${filter.type} transactions found`
```

Comment:
This also includes the number of date labels and possibly conflict headers. I suggest we filter them from the results.

Reply:
It not filters only the transactions.